entity/student.rb

```diff
@@ -16,4 +16,8 @@ def classroom=(class_room)
     @classroom = class_room
     class_room.students.push(self) unless class_room.students.include?(self)
   end
+
+  def display_person
+    '[Student]  Name: '.concat(@name).concat(' , ID: ').concat(@id.to_i).concat(' , age: ').concat(@age)
+  end
```

Comment:
- [ ] At the moment, when a new student is created and listed the ID of the student does not show which in turn makes it impossible to create a rental for that student. Please kindly fix it.

Reply:
- Well resolved 👍🏽 
I have been able to create Rental successfully!
<img width="363" alt="Screenshot 2023-02-11 at 23 22 02" src="https://user-images.githubusercontent.com/58692590/218283592-295c1132-cef0-49d8-aeaf-5d88189a733b.png">